.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - '8'
```

Comment:
Should we build on 6 as well (since you can do both with travis).  What is the minim node version this repo supports?

Reply:
_most_ versions of node 5 should work, but it should definitely work with everything 6 and up. I'll include 6, should I also include 7?

Reply:
5 and 7 are both short-lived dev versions.  We only really need to worry about the LTS releases (6 and soon to be 8) (4 is technically LTS, but almost over).